stdlib/public/Synchronization/Atomics/WordPair.swift

```diff
@@ -183,6 +183,18 @@ extension WordPair: Hashable {
   }
 }
 
+@available(SwiftStdlib 6.2, *)
+extension WordPair: Comparable {
+  // Note: This function can have a lower availability than the conformance
+  // itself because it's always emit into client.
+  @available(SwiftStdlib 6.2, *)
```

Comment:
Would the default implementations (of `>`, `<=`, and `>=`) also have a lower availability? Or would this depend on a retroactive conformance?

```suggestion
  @available(SwiftStdlib 6.0, *)
```